operations/user/usersOperation.js

```diff
@@ -0,0 +1,198 @@
+const Operation = require('../operation');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs')
+const userService = require('../../services')
+const config = require('config');
+const middleware = require('../../middleware/authenticateJWT')
+
+class UserOperation extends Operation {
+	constructor() {
+		super();
+		this.userService = userService;
+	}
+
+	async login({
+		email,
+		password
+	}) {
```

Comment:
function definition should be in one line

The objects in the next functions should also this syntax for parameters